omod/src/main/compass/sass/login.scss

```diff
@@ -6,6 +6,13 @@
   p {
     input[type=text], input[type=password] {
       width: 405px;
+      display: none;
+    }
+
+    #togglePassword{
+      position: relative;
+      margin: -50px 10px;
+      float: right;
     }
```

Comment:
@isears @ibacher when this is interpreted, it turns to 
```
#login-form p #togglePassword {
  position: relative;
  margin: -50px 10px;
  float: right;
}
```
which looks to work very fin, however, the server doesn't show the icon inside the password text field.  However, manual adding of the css from firefox inspect makes the goal attained